bedwars-plugin/pom.xml

```
@@ -198,7 +198,7 @@
         <dependency>
             <groupId>com.zaxxer</groupId>
             <artifactId>HikariCP</artifactId>
-            <version>5.0.0</version>
+            <version>3.4.5</version>
```

Why such a low version? The HikariCP's readme says that the version for Java 8 is 4.0.3
```suggestion
            <version>4.0.3</version>
```